benchmarks/docs/setup/prometheus-operator-values.yml

```diff
@@ -1,5 +1,5 @@
 alertmanager:
-  enabled: false
+  enabled: true
```

Comment:
This is for your benchmark on the long running cluster?

Reply:
Yes, but in general it'll be useful in the future to have alerts :)